web/src/main/java/no/nav/foreldrepenger/fordel/web/app/rest/JournalpostDto.java

```diff
@@ -0,0 +1,32 @@
+package no.nav.foreldrepenger.fordel.web.app.rest;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.Set;
+
+record JournalpostDto(@NotNull String journalpostId,
+                      @NotNull String tittel,
+                      Kanal kanal,
+                      @NotNull BrukerDto bruker,
+                      @NotNull AvsenderDto avsender,
+                      YtelseType ytelseType,
+                      Set<DokumentDto> dokumenter,
+                      Set<FagsakDto> fagsaker) {
+
+    record BrukerDto(@NotNull String navn, @NotNull String fnr, @NotNull String aktørId) {}
+    record AvsenderDto(@NotNull String navn, @NotNull String id) {}
+    enum Kanal { SELVBETJENING, ALLTIN, EESSI, EIA, HELSENETT, SKAN }
+    enum YtelseType { EN, FP, SVP }
+    record DokumentDto(@NotNull String dokumentId,
+                       @NotNull String tittel,
+                       @NotNull Set<Variant> varianter,
+                       @NotNull String lenke) {}
+    enum Variant { ARKIV, ORIGINAL }
+    record FagsakDto(@NotNull String saksnummer,
+                     @NotNull YtelseType ytelseType,
+                     @NotNull LocalDate datoOpprettet,
+                     @NotNull LocalDate sistEndret,
+                     @NotNull FagsakStatus status) {}
+
+    enum FagsakStatus { OPPRETTET, UNDER_BEHANDLING, LOEPENDE, AVSLUTTET }
```

Comment:
Kunne vi gjenbrukt samme kodeverk som fpsak bruker her https://github.com/navikt/fp-sak/blob/master/behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/fagsak/FagsakStatus.java#L16 ?

Så slipper vi å ha et eget kodeverk for fpfordel i frontend

Reply:
Jeg har også opprettet en FagsakDto på min branch...så her må vi samkjøre :-)

Reply:
Ja, flott tilbakemelding - kan godt gjenbruke de kodene som vi har i fpsak. Jeg ville bare i første omgang bruke primitive enums - er ikke helt glad i kodeverk opplegg. 

FagsakDto tar jeg bare i bruk senere Anja. Så det er bare å merge dine endringene.

Reply:
Vi kan godt bare bruke enums tror jeg, men kanskje gi de samme navn som i fpsak så kan vi gjenbruke litt frontend :)

Reply:
Da er kodeverk på plass. Er det bedre @pekern ?